sw/airborne/firmwares/rotorcraft/autopilot.c

```diff
@@ -576,6 +575,77 @@ bool autopilot_guided_move_ned(float vx, float vy, float vz, float heading)
   return false;
 }
 
+/* Set guided mode setpoint
```

Comment:
please use doxygen style comments so it actually ends up in the generated documentation....